EIPS/eip-721.md

```diff
@@ -440,7 +440,7 @@ XXXXERC721, by William Entriken -- a scalable example implementation
 1. Curio Cards. https://mycuriocards.com
 1. Rare Pepe. https://rarepepewallet.com
 1. Auctionhouse Asset Interface. https://github.com/dob/auctionhouse/blob/master/contracts/Asset.sol
-1. OpenZeppelin SafeERC20.sol Implementation. https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/ERC20/SafeERC20.sol
```

Comment:
Thank you for the contribution! 
This is a historical doc, to fix it
1. Editors are working on a link policy per EIP-5757 (#5757 ), let's wait for that to be finalized
2. Links are generally suggested to use permlink to reduce chance of such broken links in the future

But still, thank you for your contribution! Greatly appreciated

Reply:
@xinbenlv Yes. Exactly. This EIP is referenced by many people and we should wait for the Editors reply as it is a change to a part of that document.
I too recommend using permlink, but since the source of the change points to the default branch, I made the change using the same policy.

Reply:
> Links are generally suggested to use permlink to reduce chance of such broken links in the future

So, in this case, would the better fix be to replace the broken link with a permlink to  the last SafeERC20, rather than a link to ERC721?

And, given these links are no normative, and this one is not discussed in the proposal, I'd be happy to see it removed.